pyfolio/timeseries.py

```diff
@@ -678,20 +735,27 @@ def perf_stats(returns, factor_returns=None, positions=None):
 
     stats = pd.Series()
     for stat_func in SIMPLE_STAT_FUNCS:
-        stats[stat_func.__name__] = stat_func(returns)
+        stats[STAT_FUNC_NAMES[stat_func.__name__]] = stat_func(returns)
 
     if positions is not None:
-        stats['gross_leverage'] = gross_lev(positions).mean()
+        stats['Gross leverage'] = gross_lev(positions).mean()
+        if transactions is not None:
+            stats['Daily turnover'] = get_turnover(positions, transactions,
+                                                   period='1D').mean()
     if factor_returns is not None:
         for stat_func in FACTOR_STAT_FUNCS:
-            stats[stat_func.__name__] = stat_func(returns,
-                                                  factor_returns)
+            res = stat_func(returns, factor_returns)
+            stats[STAT_FUNC_NAMES[stat_func.__name__]] = res
+
+    for stat, value in stats.iteritems():
+        if stat in STAT_FUNCS_PCT:
+            stats[stat] = str(np.round(value * 100, 1)) + '%'
```

Comment:
I think we should do this in `show_perf_stats()` and leave these values float. Otherwise, if someone wants to do further computation with the output df from `perf_stats()` (like subtracting) it will fail for odd reasons.